modules/security-context-constraints-psa-payload-namespaces.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-security-context-constraints.adoc
+
+:_content-type: REFERENCE
+[id="security-context-constraints-psa-payload-namespaces_{context}"]
+= List of payload namespaces
+
+The following list of payload namespaces are created by your cluster. 
```

Comment:
/hold
I don't think we should keep the list anywhere unless we've got a person who maintains its accuracy. I refuse to push the responsibility on authentication documentation team. Please remove.

For the record, you're already missing:
`default`
`kubernetes-system`
`kubernetes-public`
`openshift`

If you want to have this in a module, it should nowhere mention any reference to any of the authentication-related topics (e.g. name "security-context-constraints" or "psa") and when you want to introduce it, you need to prove that there exists a process that makes documentation team take responsibility on this list being updated with any addition to payload namespaces. Also, you need to differentiate this list based on each product.

This PR will not merge until the above process is proven to exist or this section is removed completely.

Reply:
@stlaz I understand your concern. 

I think we can describe that the OCP system namespaces are: 
All namespaces prefixed with "openshift-" created in the installation  with the expectation "openshift-operators". Also, the namespace default, kubernetes-system, kubernetes-public. 

So, it is eaiser to keep maintained and will not get outdated. also, it is accurate

@maxwelldb ^

Reply:
I'm fine with this. Docs wouldn't maintain such a list in any way beyond the normal bug fix process.

Reply:
> All namespaces prefixed with "openshift-" created in the installation with the expectation "openshift-operators". Also, the namespace default, kubernetes-system, kubernetes-public.

@camilamacedo86 sounds good :+1:

I see I messed up the names, it's `kube-system`, `kube-public`

Reply:
And `kube-node-lease`, apparently